examples/annotated-skaffold.yaml

```diff
@@ -10,7 +10,7 @@ build:
     gitCommit: {}
 
     # Tag the image with the checksum of the built image (image id).
-    sha256: {}
+    # sha256: {}
```

Comment:
did you mean to comment this out?

Reply:
Yeah, this becomes invalid now. You can't define two tag policies with oneOf.

Reply:
Oh sorry, missed the `gitCommit` tagger above it.